src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -162,6 +243,7 @@ function MetadataSidebarRedesign({
                 )}
                 {editingTemplate && (
                     <MetadataInstanceEditor
+                        areBoxAiSuggestionsEnabled={areBoxAiSuggestionsEnabled}
```

Comment:
does this prop exist in the `MetadataInstanceEditor`? also we already have a prop named `isBoxAiSuggestionsEnabled` wouldn't having a prop named `areBoxAiSuggestionsEnabled ` like this be redundant and being kinda confusing?

Reply:
It's meant to be `areAiSuggestionsAvailable`, let me fix